testing_and_setup/compass/manage_regression_suite.py

```diff
@@ -417,7 +456,7 @@ def summarize_suite(suite_tag):  # {{{
 # }}}
 
 
-if __name__ == "__main__":
+def main ():  # {{{
```

Comment:
Making `main()` a function to avoid having module-level variables (sometimes prevents bugs).